src/sentry/eventstream/snuba.py

```diff
@@ -355,24 +395,24 @@ def _send(
             )
             if resp.status != 200:
                 raise snuba.SnubaError("HTTP %s response from Snuba!" % resp.status)
-            return resp
+            return None
```

Comment:
This brings the Snuba eventstream backend (used for dev/CI only) in line with the Kafka backend used in prod, which doesn't return anything